styling.css

```diff
@@ -0,0 +1,134 @@
+body {
+    margin-top:100px;
+    margin: 0;
+    padding: 0;
+    font-family "Helvetica", Arial, sans-serif;
+    background-color: honeydew;
+    }
+
+nav {
+        
+        width:100%;
+        position: flex;
+        background-color: gray;
+        font-size: 140%;
+    }
+
+nav ul {
+    list-style: none;
+    margin: 0;
+    padding: 20px;
+    display: flex;
+    }
+
+nav a {
+        color: coral;
+        padding: 15px;
+        text-decoration: none;
+        margin: 25px;
+    }
+
+
+h1 {
+    font-size:3rem;
+    font-weight: normal;
+    text-align: center;
+}
+
+h2 {
+    text-align: center;
+}
+h3 {
+    text-align: center;
+    margin-left: 20;
+    font-weight: bold;
+}
+
+em {
+    color: yellowgreen;
+    background-color: grey;
+}
+
+.abstand {
+    margin-top: 80px;
+}
+
+.mehrspaltig {
+    width: 95%;
+    margin:auto;
+    -webkit-column-width: 22rem;
+    column-width: 22rem;
+    column-count: 2;
+    column-gap: 2em;
+    column-rule-width: 1em;
+    column-rule-style:dashed;
+    column-rule-color: grey;
+}
+
+.imgPosition {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 20px;
+}
+.bildlinks{
+    float:left;
+    margin:right;
```

Comment:
margin erwartet zwischen 2 bis 4 werte für oben, rechts, unten, links und diese in px, %, rem, em, ...
mit right wird der browser nichts anfangen können